recipes/jupyterlab-lsp/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "jupyter-lsp" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3abe20ec48a3f6b132653abe597af2b6cd0005539ab0d74e0ba203927124d333
+
+build:
+  noarch: python
+  number: 0
+
+outputs:
+  - name: {{ name }}
+    script: build_base.bat  # [win]
+    script: build_base.sh  # [not win]
+    requirements:
+      host:
+        - python
+        - pip
+      run:
+        - notebook >=4.3.1
+        - python
+        - setuptools
+    test:  # Seems ignored by conda-build
+      requires:
+        - jupyterlab =2
+        - nodejs >=8,<13
+      commands:
+        - jupyter labextension install @krassowski/jupyterlab-lsp@^1.0.0
+        - python -m jupyterlab.browser_check
+  - name: {{ name }}-python
+    build:
+    requirements:
+      run:
+        - {{ pin_subpackage(name, exact=True) }}
+        - python-language-server
```

Comment:
So here, I think we'd want at least a bottom pin: I'd have to look at the build logs, but we don't matrix across `pyls` versions.

There may also be some incidentals like `parso`, `jedi`, etc. that we'd want to be more consistent about... might also be an upstream docs/ci task...